pkg/utils/tekton/controller.go

```diff
@@ -841,3 +842,40 @@ func (s *SuiteController) GetTask(name, namespace string) (*v1beta1.Task, error)
 	}
 	return &task, nil
 }
+
+func (s *SuiteController) StorePipelineRuns(componentPipelineRun *v1beta1.PipelineRun, testLogsDir, testNamespace string) error {
+	pipelineRunsDir := fmt.Sprintf("%s/pipeline-runs", testLogsDir)
+	if err := os.MkdirAll(pipelineRunsDir, os.ModePerm); err != nil {
+		return err
+	}
+
+	filepath := fmt.Sprintf("%s/%s-pr-%s.log", pipelineRunsDir, testNamespace, componentPipelineRun.Name)
+	pipelineLogs, err := s.GetPipelineRunLogs(componentPipelineRun.Name, testNamespace)
+	if err != nil {
+		g.GinkgoWriter.Printf("got error fetching PR logs: %v\n", err.Error())
```

Comment:
I think we should return an error here. WDYT?

Reply:
If we do so, we won't be able to collect any other pipelineRun logs from the namespace. But my assumption we would be able to collect other logs may be wrong. Do you think we should return an error anyway?

Reply:
If the goal is to collect all the logs possible, I think there is no need to return the error so.

I was just a little confused because if GetPipelineRunLogs fails, why try to write them?

Reply:
Yes, you are right. I will fix that.